go/rtl/auth.go

```diff
@@ -112,21 +112,25 @@ func (s *AuthSession) Do(result interface{}, method, ver, path string, reqPars m
 	if res.StatusCode < 200 || res.StatusCode > 226 {
 		b, err := ioutil.ReadAll(res.Body)
 		if err != nil {
-				return fmt.Errorf("response error. status=%s. error parsing error body", res.Status)
+			return fmt.Errorf("response error. status=%s. error parsing error body", res.Status)
 		}
 
 		return fmt.Errorf("response error. status=%s. error=%s", res.Status, string(b))
 	}
 
+	if res.StatusCode == 204 { // for delete endpoints there's no response body
+		return nil
+	}
```

Comment:
This change is the essential. The others comes from `go fmt`.

Reply:
Can you change the comment to "204 No Content. DELETE endpoints returns response with no body"?